packages/gatsby-source-mongodb/README.md

```diff
@@ -26,19 +26,47 @@ Source plugin for pulling data into [Gatsby](https://github.com/gatsbyjs) from M
 * **auth**: the authentication data to login a Mongodb collection, with sub properties user and password.
       ex. auth: { user: `admin`, password: `12345` } 
 
-### TODO WIP
+### Advanced feature
```

Comment:
Since there's only one feature, perhaps just name this header something like "Map fields for automatic transformation"